app/models/requests/SignedInUser.scala

```diff
@@ -16,14 +16,12 @@
 
 package models.requests
 
+import models.AuthKey
 import uk.gov.hmrc.auth.core.authorise.Predicate
 import uk.gov.hmrc.auth.core.{Enrolment, Enrolments}
 
 case class SignedInUser(eori: String, enrolments: Enrolments)
 
 object SignedInUser {
-
-  val cdsEnrolmentName: String = "HMRC-CUS-ORG"
-  val eoriIdentifierKey: String = "EORINumber"
-  val authorisationPredicate: Predicate = Enrolment(cdsEnrolmentName)
+  val authorisationPredicate: Predicate = Enrolment(AuthKey.enrolment)
```

Comment:
Is the entire object (`object SignedInUser`) still really necessary?

Reply:
I think so. It is the representation of a specific user who has signed in (maybe with an enrollment that does not equal "HMRC-CUS-ORG")

Reply:
But the predicate is just used to call the Auth service. It has nothing to do with the user representation. Check where `authorisationPredicate` is used, the user is not logged in yet.

Reply:
Ah, I see. Sorry, I thought you were referring to the case class `SignedInUser` rather than the companion object. Yes you are correct. I'll remove it :+1: